XCUITests/PrivateBrowsingTest.swift

```diff
@@ -5,10 +5,10 @@
 import XCTest
 
 let url1 = "www.mozilla.org"
-let url2 = "www.yahoo.com"
+let url2 = "people.mozilla.org"
 
 let url1Label = "Internet for people, not profit — Mozilla"
-let url2Label = "Yahoo"
+let url2Label = "People of Mozilla"
```

Comment:
For some reason on BB when loading yahoo.com and the tab tray view is open, instead of showing Yahoo, it sometime shows Yahoo! and so the test fail. This does not happen locally. Changing to a more stable web site in terms of what we expect it to show.